[2024-12-26T04:44:50.747+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T04:44:50.772+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transform_forex_data_dag.transform_and_load_forex_data manual__2024-12-26T04:44:48.132042+00:00 [queued]>
[2024-12-26T04:44:50.785+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transform_forex_data_dag.transform_and_load_forex_data manual__2024-12-26T04:44:48.132042+00:00 [queued]>
[2024-12-26T04:44:50.786+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-26T04:44:50.811+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): transform_and_load_forex_data> on 2024-12-26 04:44:48.132042+00:00
[2024-12-26T04:44:50.822+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=481) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-26T04:44:50.824+0000] {standard_task_runner.py:72} INFO - Started process 483 to run task
[2024-12-26T04:44:50.825+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'transform_forex_data_dag', 'transform_and_load_forex_data', 'manual__2024-12-26T04:44:48.132042+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/flatten.py', '--cfg-path', '/tmp/tmpytyvua_o']
[2024-12-26T04:44:50.828+0000] {standard_task_runner.py:105} INFO - Job 61: Subtask transform_and_load_forex_data
[2024-12-26T04:44:50.897+0000] {task_command.py:467} INFO - Running <TaskInstance: transform_forex_data_dag.transform_and_load_forex_data manual__2024-12-26T04:44:48.132042+00:00 [running]> on host 4fe538676cbc
[2024-12-26T04:44:51.028+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='transform_forex_data_dag' AIRFLOW_CTX_TASK_ID='transform_and_load_forex_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-26T04:44:48.132042+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-26T04:44:48.132042+00:00'
[2024-12-26T04:44:51.030+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T04:44:51.049+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.4, Python Version: 3.12.8, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-12-26T04:44:51.051+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2024-12-26T04:44:51.052+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-12-26T04:44:51.790+0000] {cursor.py:1166} INFO - Number of results in first chunk: 0
[2024-12-26T04:45:07.998+0000] {logging_mixin.py:190} INFO - Inserted batch 1
[2024-12-26T04:45:22.135+0000] {logging_mixin.py:190} INFO - Inserted batch 2
[2024-12-26T04:45:35.987+0000] {logging_mixin.py:190} INFO - Inserted batch 3
[2024-12-26T04:45:35.988+0000] {logging_mixin.py:190} INFO - Data transformation and insertion completed.
[2024-12-26T04:45:35.989+0000] {connection.py:789} INFO - closed
[2024-12-26T04:45:36.035+0000] {connection.py:795} INFO - No async queries seem to be running, deleting session
[2024-12-26T04:45:36.196+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-26T04:45:36.223+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-26T04:45:36.224+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=transform_forex_data_dag, task_id=transform_and_load_forex_data, run_id=manual__2024-12-26T04:44:48.132042+00:00, execution_date=20241226T044448, start_date=20241226T044450, end_date=20241226T044536
[2024-12-26T04:45:36.325+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-26T04:45:36.347+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-26T04:45:36.353+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
